pages/wishlistPage/wishlistPage.css

```diff
@@ -0,0 +1,56 @@
+@import "/styles.css";
+
+.page-wrap {
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_footer p_footer p_footer p_footer p_footer p_footer';
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.main-content_wishlist-holder {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
suggestion: you can use "repeat(3, 1fr)" here

Reply:
I'll take this as part of refactoring, due to the learning curve right now I am writing things explicitly.